doc/playground.lua

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env tarantool
+
+-- How to run:
+--
+-- $ ./doc/playground.lua
+--
+-- Or
+--
+-- $ KEEP_DATA=1 ./doc/playground.lua
+--
+-- What to do next:
+--
+-- Choose an example from README.md, doc/select.md or doc/pairs.md
+-- and run. For example:
+--
+-- tarantool> crud.select('customers', {{'<=', 'age', 35}})
+-- tarantool> crud.select('developers', nil, {first = 6})
+
+local fio = require('fio')
+local console = require('console')
+local vshard = require('vshard')
+local crud = require('crud')
+
+-- Trick to don't leave *.snap, *.xlog files. See
+-- test/tuple_keydef.test.lua in the tuple-keydef module.
+if os.getenv('KEEP_DATA') ~= nil then
+    box.cfg()
+else
+    local tempdir = fio.tempdir()
+    box.cfg({
+        memtx_dir = tempdir,
+        wal_mode = 'none',
+    })
+    fio.rmtree(tempdir)
+end
+
+-- Setup vshard.
+_G.vshard = vshard
+box.once('guest', function()
+    box.schema.user.grant('guest', 'super')
+end)
+local uri = 'guest@localhost:3301'
+local cfg = {
+    bucket_count = 3000,
+    sharding = {
+        [box.info().cluster.uuid] = {
+            replicas = {
+                [box.info().uuid] = {
+                    uri = uri,
+                    name = 'storage',
+                    master = true,
+                },
+            },
+        },
+    },
+}
+vshard.storage.cfg(cfg, box.info().uuid)
+vshard.router.cfg(cfg)
+vshard.router.bootstrap()
+
+-- Create the 'customers' space.
+box.once('customers', function()
+    box.schema.create_space('customers', {
+        format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'name', type = 'string'},
+            {name = 'age', type = 'number'},
+        }
+    })
+    box.space.customers:create_index('primary_index', {
+        parts = {
+            {field = 1, type = 'unsigned'},
+        },
+    })
+    box.space.customers:create_index('bucket_id', {
+        parts = {
+            {field = 2, type = 'unsigned'},
+        },
+    })
+    box.space.customers:create_index('age', {
+        parts = {
+            {field = 4, type = 'number'},
+        },
```

Comment:
We can add one non-unique index, I needed it a couple of times (integration tests have it).
```suggestion
        parts = {
            {field = 4, type = 'number'},
            unique = false,
        },
```

Reply:
You're right. Nice catch!

Bucket IDs, ages, names should be in a non-unique index. I fixed it.

(`unique` parameter is on the same level as `parts`, I have added it into the right place.)